README.md

```diff
@@ -125,6 +125,16 @@ all zones.
     --num-slices=4 --spot
     ```
 
+* Cluster Create for Pathways:
```

Comment:
Can we move the Pathways details to a specific section of the readme? I think it would keep the readme cleaner.

We can also give a brief description to what Pathways is as well.

Reply:
Okay, will create a Pathways section.